pkg/stream/edgedmetricsconnection.go

```diff
@@ -34,10 +34,22 @@ type EdgedMetricsConnection struct {
 	Header   http.Header   `json:"header"`
 }
 
+func (ms *EdgedMetricsConnection) CreateConnectMessage() (*Message, error) {
+	data, err := json.Marshal(ms)
+	if err != nil {
+		return nil, err
+	}
+	return NewMessage(ms.MessID, MessageTypeMetricConnect, data), nil
+}
+
 func (ms *EdgedMetricsConnection) GetMessageID() uint64 {
 	return ms.MessID
 }
 
+func (ms *EdgedMetricsConnection) String() string {
+	return fmt.Sprintf("EDGE_Metrics_CONNECTOR Message MessageID %v", ms.MessID)
```

Comment:
Should we make it "EDGE_METRICS_CONNECTOR" like other connections?

Reply:
It seems that this function is only used for log and error messages, I think it can be modified.